client/my-sites/plugins/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import DomainPicker from '@automattic/domain-picker';
+import { useShoppingCart } from '@automattic/shopping-cart';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import { domainRegistration } from 'calypso/lib/cart-values/cart-items';
+import { DomainSuggestions } from '@automattic/data-stores';
+import {
+	MARKETPLACE_FLOW_ID,
+	ANALYTICS_UI_LOCATON_MARKETPLACE_DOMAIN_SELECTION,
+} from 'calypso/my-sites/plugins/marketplace/constants';
+import { Button } from '@wordpress/components';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+import './styles.scss';
+import { getProductsList, isProductsListFetching } from 'calypso/state/products-list/selectors';
+
+type DomainSuggestion = DomainSuggestions.DomainSuggestion;
+import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+
+function Header() {
+	return (
+		<div className="marketplace-domain-upsell__header domains__header">
+			<h1 className="marketplace-domain-upsell__header marketplace-title">Choose a domain</h1>
+			<h2 className="marketplace-domain-upsell__header marketplace-subtitle">
+				Yoast SEO requires a top level domain to index your site and help you rank higher.
+			</h2>
+		</div>
+	);
+}
+
+function DomainPickerContainer( { onDomainSelect, selectedDomain } ) {
+	return (
+		<DomainPicker
+			header={ <Header /> }
+			analyticsUiAlgo={ ANALYTICS_UI_LOCATON_MARKETPLACE_DOMAIN_SELECTION }
+			analyticsFlowId={ MARKETPLACE_FLOW_ID }
+			onDomainSelect={ onDomainSelect }
+			currentDomain={ selectedDomain }
+			hideRecommendationLabel={ true }
```

Comment:
`={ true }` are redundant and can be shortened to:

```suggestion
			hideRecommendationLabel
```

Reply:
Fixed by 0aa0786